kairon/api/app/routers/bot/data.py

```
@@ -307,9 +308,33 @@
+):
+    """
+    List all collection names for the bot.
+    """
+    names = DataProcessor.get_all_collections(current_user.get_bot())
+    return Response(data=names)
+
+
+@router.delete("/collection/delete/{collection_name}", response_model=Response)
+async def delete_collection(
+    collection_name: str = Path(...),
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff (0.11.9)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)